documentation/bolt_configure_orchestrator.md

```diff
@@ -1,13 +1,16 @@
 # Connecting Bolt to Puppet Enterprise
 
-If you're a Puppet Enterprise (PE) customer, you can connect Bolt to Puppet Enterprise (PE) using the
-Puppet Communications Protocol (PCP) transport. However, in most cases this is not
-necessary, because tasks and plans are already supported from the console or the
-command line using
-[PE orchestrator](https://puppet.com/docs/pe/latest/running_jobs_with_puppet_orchestrator_overview.html). Wherever possible, we recommend using PE tasks and plans instead of connecting
+If you're a Puppet Enterprise (PE) customer, you can connect Bolt to Puppet
```

Comment:
Do we need to do the "Puppet Enterprise (PE)" bit twice?

Reply:
Oh good catch, I'll remove one